Techome/Techome/ViewModel/AddCaffeineDetailStateHolder.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AddCaffeineDetailStateHolder.swift
+//  Techome
+//
+//  Created by 한택환 on 2022/06/15.
+//
+
+import Foundation
+
+class AddCaffeineDetailStateHolder: ObservableObject {
```

Comment:
```suggestion
final class AddCaffeineDetailStateHolder: ObservableObject {
```

더이상 상속되지 않는 class라면 final이 필요합니다 ~